src/ResourceManager/EventHub/Commands.EventHub.Test/ScenarioTests/ConsumerGroupsTests.ps1

```diff
@@ -64,7 +64,7 @@ function ConsumerGroupsTests
     
     Write-Debug "  Create resource group"
     Write-Debug " Resource Group Name : $resourceGroupName"
-    $Result11 = New-AzureRmResourceGroup -Name $resourceGroupName -Location $location -Force
```

Comment:
@v-Ajnava why is this line commented out now? Does this affect the asserts below?